scripts/dev-test-rc.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
I created it for handy development purpose to run test from ide to see the log on console etc.